app/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once(__DIR__ . '/src/Db.php');
+require_once(__DIR__ . '/src/XmlParser.php');
+require_once(__DIR__ . '/src/Command.php');
+
+$url = getenv('NEXTCLOUD_IDPS_METADATA_URL');
+
+if (empty($url)) {
+  echo 'idps metadata url is missing. Set NEXTCLOUD_IDPS_METADATA_URL environment variable to provide it.';
+  exit(1);
+}
+
+$db = new Db(
+  getenv('POSTGRES_HOST'),
+  getenv('POSTGRES_PORT'),
```

Comment:
Suggestion, provide a default value at least for the port : 

```suggestion
  getenv('POSTGRES_PORT') ?: 5432,
```